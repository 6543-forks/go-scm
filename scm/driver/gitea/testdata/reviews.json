[
  {
    "body": "This is a review",
    "comments_count": 0,
    "commit_id": "5c23b301e7eb47aa83de90cf08e0a75b4c0906c8",
    "html_url": "https://try.gitea.io/jcitizen/my-repo/pulls/1/reviews/1",
    "id": 1,
    "official": true,
    "pull_request_url": "https://try.gitea.io/jcitizen/my-repo/pulls/1",
    "stale": false,
    "state": "PENDING",
    "submitted_at": "2020-09-07T16:19:57.863Z",
    "user": {
      "id": 6641,
      "login": "jcitizen",
      "full_name": "",
      "email": "jcitizen@example.com",
      "avatar_url": "https://secure.gravatar.com/avatar/66f07ff48e6a9cb393de7a34e03bb52a?d=identicon",
      "language": "en-US"
    }
  }
]
