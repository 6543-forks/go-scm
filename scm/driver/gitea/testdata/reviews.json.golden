[
  {
    "Body": "This is a review",
    "Sha": "5c23b301e7eb47aa83de90cf08e0a75b4c0906c8",
    "Link": "https://try.gitea.io/jcitizen/my-repo/pulls/1/reviews/1",
    "ID": 1,
    "State": "PENDING",
    "Created": "2020-09-07T16:19:57.863Z",
    "Author": {
      "ID": 6641,
      "Login": "jcitizen",
      "Email": "jcitizen@example.com",
      "Avatar": "https://secure.gravatar.com/avatar/66f07ff48e6a9cb393de7a34e03bb52a?d=identicon"
    }
  }
]
